db/mysql/migrate/000001_initial_schema.up.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE IF NOT EXISTS Service(ID varchar(36) PRIMARY KEY, Name VARCHAR(255));
```

Comment:
We use snake case naming for identifiers in databases.

In MySQL, the table names rely on the underlying system https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html

```suggestion
CREATE TABLE service(id char(36) PRIMARY KEY, name varchar(255));
```